internal/docstore/firedocstore/codec.go

```diff
@@ -142,7 +141,7 @@ func floatval(x float64) *pb.Value { return &pb.Value{ValueType: &pb.Value_Doubl
 ////////////////////////////////////////////////////////////////
 
 // decodeDoc decodes a Firestore document into a driver.Document.
-func decodeDoc(pdoc *pb.Document, ddoc driver.Document, nameField string) error {
+func decodeDoc(pdoc *pb.Document, ddoc driver.Document, nameField, revField string) error {
```

Comment:
I suggest wrap `nameField` and `revField` into an options struct.

Reply:
I see your point. Maybe if we add one more thing that will be worth it.